front_end/html/edit_assignment.html

```diff
@@ -247,6 +247,25 @@ <h2>Create assignment</h2>
         </p>
     </div>
 
+    <div class="shadow-box">
+        <p><strong>Restrict assignment access to the following IP addresses: </strong></br>Separate valid addresses by a comma without spaces.</p>
+        <div class="select is-grey">
+            <select id="access_restricted" onchange="showAllowedIPAddresses(this.value)" name="access_restricted" class="edit-select">
+                {% if assignment_details["allowed_ip_addresses"] is not None %}
+                <option value="Yes" selected>Yes</option>
+                <option value="No">No</option>
+                {% else %}
+                <option value="Yes">Yes</option>
+                <option value="No" selected>No</option>
+                {% end %}
+            </select>
+        </div>
+
+        {% if assignment_details["allowed_ip_addresses"] is not None %}
```

Comment:
Could you use the split() function here? I think Python functions are supported, and that would be more succinct. Also, I'm not sure if the current solution would put an extra comma at the end.

Reply:
I'm not sure I understand what you mean here. assignment_details["allowed_ip_address"] is already in an array when it's accessed by the html, do you mean to use join() inside the textarea to display current ip addresses or am I just misunderstanding how you want to use split()?